select_ppx_version.sh

```diff
@@ -4,7 +4,7 @@ V=`ocamlc -version`
 case $V in
 4.02*) SRC=402 ;;
 4.03*) SRC=403 ;;
-*) echo Unsupported OCaml version $V ;;
+*) SRC=error; echo Unsupported OCaml version $V ;;
```

Comment:
isn't an `exit 1` sufficient, rather than distributing invalid ml code for the error case?